hq_superset/views.py

```diff
@@ -247,8 +232,8 @@ def post_dataset_change(self) -> FlaskResponse:
 
         try:
             request_json = json.loads(request.get_data(as_text=True))
-            change = DataSetChange(**request_json)  # raises TypeError
-            update_dataset(change)  # raises ValueError
+            change = DataSetChange(**request_json)
+            update_dataset(change)
```

Comment:
@kaapstorm Should this be commented out?

Reply:
No? That's the call that does the work.